third_party/tonic/logging/dart_error.cc

```diff
@@ -10,18 +10,20 @@
 namespace tonic {
 namespace DartError {
 const char kInvalidArgument[] = "Invalid argument.";
-UnhandledExceptionReporter log_unhandled_exception = [](Dart_Handle,
-                                                        Dart_Handle) {};
+}  // namespace DartError
 
+namespace {
+DartError::UnhandledExceptionReporter log_unhandled_exception =
+    [](Dart_Handle, Dart_Handle) {};
```

Comment:
I don't think this is going to work, I think you'll have to assign it to a function.
```suggestion
void DefaultLogUnhandledException(Dart_Handle, Dart_Handle) {}
DartError::UnhandledExceptionReporter log_unhandled_exception = DefaultLogUnhandledException;
```

Reply:
Done.